sos/report/plugins/scsi.py

```diff
@@ -29,6 +29,10 @@ class Scsi(Plugin, IndependentPlugin):
     profiles = ('storage', 'hardware')
 
     def setup(self):
+        _subdir1 = "udevadm_info_scsihosts"
+        _subdir2 = "sg_inq_devices"
+        _subdir3 = "sg_persist_devices"
+
```

Comment:
Ditto variable names.

Reply:
>> My first thought was "why the tag has redundant _sos substring, it is evident the tag belongs to sos - but we already had fdisk_l_sos tag so I am OK,
>> For code readiness, could be the variable names be more descriptive, like e.g. udevadm_dir or fdisk_dir?
>> Ditto variable names.

Not added _dir for nvme or other commands due to similar reasons as we know that it's a directory. OR shall I change it similar to  list_ctrl_dir. Actual command is "nvme list-ctrl" hence there is nvme and its not because of the plugin. Could you please consider giving another thought on the same.